components/Recipes/index.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Recipe from "../Recipe";
+import styled from "styled-components";
+import { RecipeClass, recipes } from "../../lib/recipes";
+
+const RecipesList = styled.div`
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  justify-content: center;
+  margin: 20px;
+
+  li {
+    width: 150px;
+    min-width: 50px;
+  }
+
+  @media (min-width: 500px) {
+    li {
+      width: 300px;
+    }
```

Comment:
Nested styling like this in dangerous as this styling will always have a higher specificity than every other explicitly StyledLi you could place within the RecipeList. It is better to have a separate styled-component for the li's.

Reply:
Thanks for the tip, I'll adjust it